### DISCUSSION TITLE
Move Multiplay Function from VRoidMultiplayCharacter To Function Library

### DISCUSSION THREAD

**Discussion by ToffelsKater** [pixiv/vroid-sdk-developers#161] [General]
Is it possible to move the necessary functions to a library so it can be used outside the VRoidMultiplayCharacter?
<img width="647" alt="image" src="https://github.com/pixiv/vroid-sdk-developers/assets/20298637/61a9dff9-a3a7-4d96-bb1a-1e8efb8b50bd">

**Comment by ToffelsKater** [comment]
Got my own implementation

**Reply by lobotomy-x** [reply]
this sounds useful would you be willing to share?
